gaphas/aspect/connector.py

```diff
@@ -88,16 +107,74 @@ def disconnect(self) -> None:
 Connector = singledispatch(ItemConnector)
 
 
+@Connector.register(Line)
+class LineConnector(ItemConnector):
+    def secondary_handle(self) -> Optional[Handle]:
+        item = self.item
+        handle = self.handle
+        handles = item.handles()
+        if len(handles) < 2:
+            return None
+        if handle is item.handles()[0]:
```

Comment:
Could this be simplified to `if handle is handles[0]`:?

Reply:
Yes of course :)

Reply:
Yes, it should :)